### PR TITLE
skill param updated. query fallbacks implemented

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
skill param was using old format, so skills weren't being passed in.

Fallbacks updated------------------
Fallbacks that were already in place:
Positions that match users’ grade and skill (if user has at least one skill and a grade in userProfile)
Positions that match users’ grade (if user has grade in userProfile)
Users’ favorite positions (grab users’ Favorites, regardless of if they have any)

But the fallbacks weren’t accounting for if the user **did** have skill and grade but the query matched no positions.

### CODE REVIEW

File: src/actions/homePagePositions.js [hunk 1]
```diff
@@ -46,10 +46,10 @@ export function homePagePositionsFetchData(skills = [], grade) {
       [HIGHLIGHTED_POSITIONS]: [],
     };
     const queryTypes = [];
-    // Search for positions that match the user's grade and skills.
-    // else search for just positions that match the user's grade.
-    // Otherwise, search for user favorites.
-
+    // arrangement:
+    // 1: userSkillAndGradePositions
+    // 2: userGradePositions
```
Comment (mjoyce91) [inline src/actions/homePagePositions.js:51]: This does not appear if the user has a grade but no skills

Comment (mjoyce91) [reply src/actions/homePagePositions.js:51]: But it works as you explained when the user has both a grade and skill, but it simply returns 0 results.

File: src/actions/homePagePositions.js [hunk 2]
```diff
@@ -46,10 +46,10 @@ export function homePagePositionsFetchData(skills = [], grade) {
...
+    // 3: favoritedPositions
```
Comment (mjoyce91) [inline src/actions/homePagePositions.js:52]: If the user has a grade and a skill, but the query for those returns 0 results, you'll briefly see "No results found", and then it switches to Favorites.

Comment (elizabeth-jimenez) [reply src/actions/homePagePositions.js:52]: fixed, but not sure if it's best practice

**Pull request merged**